server/views/partials/layout.njk

```diff
@@ -125,9 +123,11 @@
 {% block bodyEnd %}
   {# Run JavaScript at end of the
   <body>, to avoid blocking the initial render. #}
+  <script src="/assets/js/jquery.min.js"></script> 
   <script src="/assets/govuk/all.js"></script>
   <script src="/assets/govukFrontendInit.js"></script>
   <script src="/assets/moj/all.js"></script>
+  <script src="/assets/mojukFrontendInit.js"></script>
```

Comment:
Similar to how the `govukFrontendInit` JavaScript is initialised above, the MoJ frontend JavaScript needed initialising for the [Sortable table component](https://design-patterns.service.justice.gov.uk/components/sortable-table/) to work